DOCUMENTATION.md

```diff
@@ -2733,7 +2733,11 @@ You can configure this using `metadata-policy` element for the map configuration
 
 #### 8.8.1.5. Filtering with Paging Predicates
 
+> **WARNING: Starting from version 4.2, [the SQL querying feature](#87-sql) is introduced.
+> The SQL querying feature is more efficient than paging predicates, and it is recommended to be used instead of paging predicates.**
+
 The Node.js client provides paging for defined predicates. With its `PagingPredicate` object, you can get a list of keys, values or entries page by page by filtering them with predicates and giving the size of the pages. Also, you can sort the entries by specifying comparators.
+You can find an example code [here.](https://github.com/srknzl/hazelcast-paging-predicate-example)
```

Comment:
I am not okay with having some links pointing to the code sample at a place that is not in control of Hazelcast, for Hazelcast features. I mean, there is no guarantee that you won't delete it in the future :smile: So, I would leave the documentation as it is.

BTW, why do you used some custom comparator for integer values? Can't it be more suitable if you show some sort of complex object, which cannot be used out-of-the-box with paging predicates?

Reply:
we can move the repository to hazelcast organization. Should I use a complex object to showcase how it is used? Why it is more suitable? Also, see the new reported issue from Andrey #1009, is the sort method redundant in nodejs client. If so I can remove it in this pr.

Reply:
Yeah, that method is redundant and I feel like can be removed in 5.0. But, I feel like we can do it another PR.

I feel like if we are going such long measures, let's build something more complex BUT I don't feel like this feature (paging predicate) deserves such thing. Was there a user request for it, for the extended documentation?

Reply:
Yes someone asked how he can use the paging predicate with sorting. And I prepared a the repo for him. That's why I opened this PR. 

Then, I told him that he can use SQL query which is easier and he was happy to hear it. However, I think for legacy users we can include such a guide.

Reply:
Then, maybe I would provide this repo you prepared as a code sample in this repo, similar to the JAAS sample, with its own README etc. explaining the necessary steps to run it, BUT that is definitely not a must.

Like I said, I don't think we need to promote this feature more as it is highly inefficient

Reply:
I have already added a warning about its unefficiency against sql. I think it fine to add a sample for legacy users.